### ISSUE TITLE
ipla.tv

### ISSUE THREAD

**Issue by mareczek1** [MajkiIT/polish-ads-filter#8267]
### Zrzut ekranu

![image](https://user-images.githubusercontent.com/40866551/42335901-2d5ab67a-8082-11e8-99a0-9d77fb29f403.png)

`https://www.ipla.tv/wideo/sport/Moto-GP/5013581`

### Co trzeba zrobić, aby pojawił się element, reklama albo błąd?
Po włączeniu strony wykrywa blokowanie reklam

### Moja konfiguracja
**Przeglądarka internetowa:**
Google chrome
Wersja 67.0.3396.99 (Oficjalna wersja) (64-bitowa)
**Rozszerzenia i skrypty:**
uBlock Origin 1.16.12, Anti-adblock killer | Reek, Nano Defender 15.0.0.4
**Filtry:**

![image](https://user-images.githubusercontent.com/40866551/42336370-cd964b80-8083-11e8-995b-1cdbbcda3298.png)

**Comment by KonoromiHimaries** [comment]
skrypt `Anti-adblock killer | Reek` możesz usunąć gdy używasz nano defendera

**Comment by MajkiIT** [comment]
Na FF jest OK. Pewnie zaraz się okarze, że to wina tego samego co na Player.pl

**Comment by mareczek1** [comment]
2 dni temu działało normalnie.

**Comment by FadeMind** [comment]
U mnie działa
![20180705-005-screenshot](https://user-images.githubusercontent.com/9846948/42338660-d845a8ee-808a-11e8-8da3-73b7740f290c.png)


	Information                         | Value
	---                                 | ---
	Operating system:                   | Windows 10 1803 x64
	Browser:                            | Chrome Stable x64
	AdGuard version:                    | **6.3.1374.4023** RC
	Stealth mode: 						| enabled

	| Filters enabled |
	|-----------------|
	| Adguard Annoyances 
	| Adguard English
	| Adguard Social
	| Adguard Spy 
	| AlleBlock
	| EasyList Polish 
	| Fanboy's Annoyances
	| Fanboy's Enhanced Tracking List
	| Fanboy's Social Blocking List
	| I don't care about cookies
	| KAD - fraud/scam 
	| Malware Domains List
	| NoCoin Filter List 
	| Peter Lowe's list 
	| Polish Adguard Suplement
	| Polish Annoyances Adguard Suplement
	| Polish Annoyances Main 
	| Polish Cookies Filter
	| Polish Main
	| Polish Social
	| Spam404

**Comment by hawkeye116477** [comment]
U mnie tak samo działa

**Comment by KonoromiHimaries** [comment]
@FadeMind 

> ** Jeżeli korzystasz z Polskich Filtrów Elementów Irytujących, to nie dodawaj filtrów RODO-Ciasteczkowych. Ponieważ filtry RODO-Ciasteczkowe są już zawarte w filtrach Elementów Irytujących.

**Comment by FadeMind** [comment]
@KonoromiHimaries  Adguard nie honoruje póki co w pełni funkcji INCLUDE. 
https://github.com/PolishFiltersTeam/PolishAnnoyanceFilters/issues/956#issuecomment-393634438

**Comment by hawkeye116477** [comment]
Ale RODO-Ciasteczkowe są zarówno na głównej liście EI jak i supplemencie, bo RODO-Ciasteczkowe też mają supplement i oczywiście RODO-Ciasteczkowe są dołączane nie instrukcją #include, a skryptem :smile:.

**Comment by MajkiIT** [comment]
@mareczek1 nadal masz problem ?

**Comment by mareczek1** [comment]
@MajkiIT Tak, dalej jest tak samo. Nic nie kombinowałem bo imo wszystko w ustawieniach ublocka jest jak powinno. Adgurda nie mam i w sumie nie wiem co to/za ile więc sobie go odpuszczam.
